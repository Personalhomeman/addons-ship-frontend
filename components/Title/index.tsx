import { ReactNode } from 'react';

import css from './style.scss';

interface TitleProps {
  children: ReactNode;
}

export default ({ children }: TitleProps) => {
  return <h1 className={css.title}>{children}</h1>;
};
