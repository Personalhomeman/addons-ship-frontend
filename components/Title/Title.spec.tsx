import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Title from '.';

describe('Title', () => {
  it('renders correctly', () => {
    const tree = toJSON(shallow(<Title>Some title</Title>));
    expect(tree).toMatchSnapshot();
  });
});
