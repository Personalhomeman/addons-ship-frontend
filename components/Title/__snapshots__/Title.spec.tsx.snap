// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<h1
  className="title"
>
  Some title
</h1>
`;
